### DISCUSSION TITLE
What should be the address and port in the DOCKER_HOST_PATH to communicate with the Docker daemon via Plaintext in Bottlerocket?

### DISCUSSION THREAD

**Discussion by effy-coding** [bottlerocket-os/bottlerocket#2973] [Q&A]
When using dockershim.sock in Bottlerocket, I want to use it without TLS. What should be the address and port in the DOCKER_HOST_PATH to communicate with the Docker daemon via Plaintext in Bottlerocket?

For reference, I'm running Jenkins' Docker in Docker model on Bottlerocket, with the host volume mounted as follows

volumes:
            - type: HostPath
              hostPath: /var/run/dockershim.sock
              mountPath: /var/run/docker.sock

**Comment by effy-coding** [comment]
I used port 2375 and got the following error 

Cannot connect to the Docker daemon at tcp://localhost:2375. Is the docker daemon running?

**Comment by jpculp** [comment]
This is a shot in the dark, but do you get the same behavior if you mount to `/run` instead of `/var/run`? _(Just want to rule out `selinux`.)_

Also is this the `ecs` variant or `k8s`? Docker is only available in our `ecs` variants, whereas `k8s` variants use `containerd`.
